app/Http/ViewHelpers/Company/HR/CompanyHRViewHelper.php

```diff
@@ -30,9 +30,7 @@ public static function eCoffees(Company $company): ?array
         $allMatchesInCompany = DB::table('e_coffees')
             ->where('company_id', $company->id)
             ->select('id')
-            ->get()
-            ->pluck('id')
-            ->toArray();
+            ->pluck('id');
```

Comment:
Same comment as above. Pluck returns a collection.

Reply:
Laravel will convert it to an array when required.